README.rst

```diff
@@ -10,27 +10,28 @@
 .. image:: https://img.shields.io/codecov/c/github/pypa/twine
     :target: https://codecov.io/gh/pypa/twine
 
-.. DO-NOT-REMOVE-docs-badges-END
```

Comment:
Why don't you like DONTREMOVE markers? I'm usually using the uppercased ones specifically to urge people not knowing what they do, not to remove them in their refactoring...